docs/releases/v0.3.0.md

```diff
@@ -0,0 +1,292 @@
+
+Welcome to the latest release of `kafka` module of the [`SIGHUP Distribution`](https://github.com/sighupio/distribution) maintained with ❤️ by team [SIGHUP by ReeVo](https://sighup.io/).
+
+## Component Images 🚢
+
+| Component                | Supported Version                                                                                      | Previous Version |
+|--------------------------|--------------------------------------------------------------------------------------------------------|------------------|
+| `Strimzi Operator`       | [`v0.45.0`](https://github.com/strimzi/strimzi-kafka-operator/releases/tag/0.45.0)                     | `v0.42.0`        |
+| `Monitoring configs`     | `v0.3.0`                                                                                               | `v0.2.0`         |
+
+> Please refer to the individual release notes to get detailed information on each release.
+
+## Bug Fixes and Changes 🐛
+
+- Strimzi operator upgraded to `v0.45.0`
+- Add support for Kafka versions `3.8.0`, `3.8.1`, `3.9.0`
+
+## Breaking Changes 💔
+
+- Drop support for Kafka versions < `3.8.0`
+- Drop support for ZooKeeper
+- Drop support for Kafka cluster not using `KafkaNodePool`
+
+## Update Guide 🦮
+
+The upgrade procedure consists in four steps of which the first two needs to be done only if the Kafka cluster already existing don't use KRaft and KafkaNodePools:
+
+1. Migrate existing Kafka cluster to KafkaNodePools (Optional)
+2. Migrate from ZooKeeper to KRaft (Optional)
+3. Upgrade the operator.
+4. Upgrade the Kafka cluster.
+
+
+### 1. Migrate existing Kafka cluster to KafkaNodePools
+
+If already existing Kafka clusters do not use KafkaNodePools then this step is necessary to proceed with the upgrade. First a `KafkaNodePool` resource needs to be created. The resource should be labeled with the `strimzi.io/cluster: <CLUSTER_NAME>` label where `<CLUSTER_NAME>` is the Kafka resource name. Set the replica count and storage configuration to match your current Kafka cluster, also set the roles to broker. Here is an example manifest:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaNodePool
+metadata:
+  name: kafka
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  replicas: 3
+  roles:
+    - broker
+  storage:
+    type: jbod
+    volumes:
+      - id: 0
+        type: persistent-claim
+        size: 10Gi
+        deleteClaim: false
+```
+
+And here is an example Kafka resource manifest:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+  zookeeper:
+    # ...
+```
+
+> [!WARNING]  
+> To preserve cluster data and the names of its nodes and resources, the node pool name must be kafka, and the strimzi.io/cluster label matches the Kafka resource name. Otherwise, nodes and resources are created with new names, including the persistent volume storage used by the nodes. Consequently, your previous data may not be available.
```

Comment:
```suggestion
> To preserve cluster data and the names of its nodes and resources, the node pool name must be `kafka`, and the `strimzi.io/cluster` label must match the Kafka resource name. Otherwise, nodes and resources will be created with new names, including the persistent volume storage used by the nodes, and consequently your previous data may not be available.
```